doc/advanced.rst

```diff
@@ -136,6 +136,10 @@ Creating a filter is as simple as associating a name with a PHP callable::
     // or a simple PHP function
     $filter = new Twig_SimpleFilter('rot13', 'str_rot13');
 
+    // or a class static method
+    $filter = new Twig_SimpleFilter('rot13', array('SomeClass', 'rot13Filter'));
```

Comment:
This is indeed the exact same notation but one would be a static method while the other one is not. I've added some notes to make it (I hope) more clear.